CMakeLists.txt

```diff
@@ -257,6 +257,40 @@ configure_file(programs/cleos/LICENSE.CLI11                                licen
 
 install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/licenses/leap" DESTINATION "${CMAKE_INSTALL_FULL_DATAROOTDIR}/licenses/" COMPONENT base)
 
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/libraries/testing/contracts DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/libraries/testing COMPONENT dev
+        PATTERN "CMakeFiles" EXCLUDE
+        PATTERN "*.cmake" EXCLUDE
+        PATTERN "Makefile" EXCLUDE)
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/unittests/contracts DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/unittests COMPONENT dev
+        PATTERN "CMakeFiles" EXCLUDE
+        PATTERN "*.cmake" EXCLUDE
+        PATTERN "Makefile" EXCLUDE)
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/tests/TestHarness DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/tests COMPONENT dev
+        FILES_MATCHING
+        PATTERN "*.py"
+        PATTERN "*.json"
+        PATTERN "__pycache__" EXCLUDE
+        PATTERN "CMakeFiles" EXCLUDE)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/tests/launcher.py DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/tests COMPONENT dev)
+
+if(${CMAKE_VERSION} GREATER 3.22)
```

Comment:
Probably should use `VERSION_GREATER` here (or maybe even `VERSION_GREATER_EQUAL` to make it a little clearer); also no need to expand the variable.

Can you explain a little more what the problem is for pre vs post 3.22 (3.23?)? For example, you only set the `CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA` in the pre-3.22/3 path; is it really unneeded in the post-3.22/3 path? Is it really not possible to just use the pre-3.22/3 path on all versions?

Reply:
Just double checking, you're suggesting:

`if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.22)`

Yeah I can get on board with that, it is clearer.

Reply:
> Can you explain a little more what the problem is for pre vs post 3.22 (3.23?)? For example, you only set the `CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA` in the pre-3.22/3 path; is it really unneeded in the post-3.22/3 path? Is it really not possible to just use the pre-3.22/3 path on all versions?

Because of the use of `install(CODE` and the false error reporting that comes out of that piece during a `make package`, was hoping to leave the >3.22 path in place with the hopes of remove the <3.22 path when we get to a point where we aren't supporting older versions of cmake where the `create_symlink` function has yet to be fixed.  And also alleviate that reporting for anyone building/installing on newer cmake toolchains.

Reply:
Yes but `if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.22)` may be different then `if(${CMAKE_VERSION} GREATER 3.22)` because the original might not have been true for 3.22.0 since 3.22.0 might not have been greater than 3.22. So double check which version you really want here maybe it's 3.22 or 3.23

Reply:
So it is actually due to cmake (fixed in 3.21) not allowing symbolic links to a directory to be installed via `install(FILES`

Cmake 3.21 fixed this (https://gitlab.kitware.com/cmake/cmake/-/commit/d71a7cc19d6e03f89581efdaa8d63db216184d40)
`Allow symbolic links to a directory to be installed via install(FILES ...)`

Reply:
Addressed: [be154da](https://github.com/AntelopeIO/leap/pull/818/commits/be154da6036dbe810631b53b32afee58491767fc)